google-cloud-clients/google-cloud-websecurityscanner/src/test/java/com/google/cloud/websecurityscanner/it/v1beta/VpcPositiveTest.java

```diff
@@ -0,0 +1,270 @@
+  private static final String IN_VPCSC_GOOGLE_CLOUD_TEST_ENV = "GOOGLE_CLOUD_TESTS_IN_VPCSC";
+  private static final String IN_VPCSC_PROJECT_ENV =
+      "GOOGLE_CLOUD_TESTS_VPCSC_INSIDE_PERIMETER_PROJECT";
+  private static final String IN_VPCSC_HOSTNAME_ENV = "GOOGLE_CLOUD_WEBSECURITYSCANNER_HOSTNAME";
+  private static final String GOOGLE_CREDENTIAL_DEFAULT_ENV = "GOOGLE_APPLICATION_CREDENTIALS";
+  private static final String GOOGLE_API_CLOUD_PLATFORM_LINK =
+      "https://www.googleapis.com/auth/cloud-platform";
+
+  private static final String IN_VPCSC_TEST = System.getenv(IN_VPCSC_GOOGLE_CLOUD_TEST_ENV);
+  private static final String IN_VPCSC_PROJECT = System.getenv(IN_VPCSC_PROJECT_ENV);
+  private static final String IN_VPCSC_HOSTNAME = System.getenv(IN_VPCSC_HOSTNAME_ENV);
+  private static final String IN_VPCSC_GOOGLE_CREDENTIAL =
+      System.getenv(GOOGLE_CREDENTIAL_DEFAULT_ENV);
+
+  private String test0DisplayName;
+  private String test1DisplayName;
+
+  private static boolean isNotEmpty(String value) {
+    return value != null && value.length() != 0;
+  }
+
+  @BeforeClass
+  public static void setUpClass() {
+    assumeTrue(
+        "To run tests, GOOGLE_CLOUD_TESTS_IN_VPCSC environment variable needs to be set to true",
+        IN_VPCSC_TEST != null && IN_VPCSC_TEST.equalsIgnoreCase("true"));
+
+    assertTrue(
+        IN_VPCSC_PROJECT_ENV
+            + " environment variable needs to be set to a GCP "
+            + "project that is inside the VPC perimeter",
+        isNotEmpty(IN_VPCSC_PROJECT));
+
+    assertTrue(
+        IN_VPCSC_HOSTNAME_ENV
+            + " environment variable needs to be set to a web "
+            + "application that resides inside "
+            + IN_VPCSC_PROJECT,
+        isNotEmpty(IN_VPCSC_HOSTNAME));
+
+    assertTrue(
+        GOOGLE_CREDENTIAL_DEFAULT_ENV
+            + " environment variable needs to be set to "
+            + "google application credentials that resides inside VPCSC",
+        isNotEmpty(IN_VPCSC_GOOGLE_CREDENTIAL));
+  }
+
+  @Before
+  public void setUp() {
+    DateTimeFormatter formatter =
+        DateTimeFormat.fullDateTime().withZone(DateTimeZone.forID("America/Los_Angeles"));
+    String currentTime = formatter.print(DateTime.now(DateTimeZone.forID("America/Los_Angeles")));
+    test0DisplayName = "vpcsctest-" + currentTime + "-0";
+    test1DisplayName = "vpcsctest-" + currentTime + "-1";
+  }
+
+  private WebSecurityScannerSettings getWssSettingWithCredentials() throws IOException {
+    GoogleCredentials credentials =
+        GoogleCredentials.fromStream(new FileInputStream(IN_VPCSC_GOOGLE_CREDENTIAL))
+            .createScoped(Lists.newArrayList(GOOGLE_API_CLOUD_PLATFORM_LINK));
+    return WebSecurityScannerSettings.newBuilder()
+        .setCredentialsProvider(FixedCredentialsProvider.create(credentials))
+        .build();
+  }
+
+  private WebSecurityScannerSettings getWssSettingWithoutCredentials() throws IOException {
```

Comment:
I'd probably rename this to `getWssSettingsWithDefaultCredentials` as there are credentials - they are just the default detected ones from the execution environment.

Reply:
Made this change.